### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by davehunt**

### CODE REVIEW

**Comment by davehunt** [comment]
Pinging @tojonmz and @rbillings for review.

File: README.md [hunk 1]
```diff
@@ -1,79 +1,86 @@
...
+```bash
+$ py.test --baseurl https://treeherder.mozilla.org --driver firefox
+```
+
+The pytest plugin that we use for running tests has a number of advanced
+command line options available. To see the options available, run
+`py.test --help`. The full documentation for the plugin can be found
+[here][pytest-mozwebqa].
+
+[contributors]: https://github.com/mozilla/Addon-Tests/contributors
```
Comment (stephendonner) [inline README.md]: This is the repo for Addon-Tests

File: README.md [hunk 2]
```diff
@@ -1,79 +1,86 @@
...
+# Tests for Mozilla's Treeherder
+This repository contains tests for [Mozilla's Treeherder](https://treeherder.mozilla.org/).
+
+[![license](https://img.shields.io/badge/license-MPL%202.0-blue.svg)](https://github.com/mozilla/treeherder-tests/blob/master/LICENSE)
+[![travis](https://img.shields.io/travis/mozilla/treeherder-tests.svg?label=travis)](http://travis-ci.org/mozilla/treeherder-tests/)
+[![stage](https://img.shields.io/jenkins/s/https/webqa-ci.mozilla.com/treeherder.stage.svg?label=stage)](https://webqa-ci.mozilla.com/job/treeherder.stage/)
+[![requirements](https://img.shields.io/requires/github/mozilla/treeherder-tests.svg)](https://requires.io/github/mozilla/treeherder-tests/requirements/?branch=master)
+
+## Getting involved
+We love working with contributors to fill out the test coverage for Marketplace,
```
Comment (stephendonner) [inline README.md]: s/Marketplace/Treeherder, here

**Comment by davehunt** [comment]
Thanks @stephendonner! Comments addressed.

**Comment by tojonmz** [comment]
@davehunt I noticed Ed switched to the long form MPL in the treeherder repo itself in https://github.com/mozilla/treeherder/commit/ad2048b83ff82e85d15d4d10774571470b66346a as part of [PR850](https://github.com/mozilla/treeherder/pull/850). Just in case it's of interest or applicable for testing repos.

**Comment by davehunt** [comment]
Thanks @tojonmz, however after re-reading through section 1.4 and exhibit A of https://www.mozilla.org/en-US/MPL/2.0/ I think it's fine to just use the short form. It's also generally better to include the the license in all files:

> While the license permits putting the header somewhere other than the file itself, individual files often end up being distributed on their own, without the rest of the software they were authored with. As a result, putting the license notice in the file is the surest way to ensure that recipients are always notified.

From https://www.mozilla.org/en-US/MPL/2.0/FAQ/

**Comment by davehunt** [comment]
Also pinging @camd for visibility/review.

**Comment by rbillings** [comment]
r+ :cat:

**Pull request merged**